src/Variant/RandomChooser.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace PhpAb\Variant;
+
+class RandomChooser implements ChooserInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function chooseVariant($variants)
+    {
+        $chosen = array_rand($variants);
```

Comment:
I'm not sure whats faster.
I want to keep the possibility to use assoc array here.

Thiswhile I would have to call

```
$keys = array_keys($array);
$key = $keys[1];
```

WDYT? Better randomness should not be needed here.